PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+#Azure SDK for Java Pull Request Template Guide
+
+Thank you for your interest in contributing to Azure SDK for Java.
+
+## Description
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
+List any dependencies that are required for this change.
+
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
```

Comment:
It's weird to say 'please delete' and then give a check list of options to select.